engine/kv_engine.cpp

```diff
@@ -846,8 +848,8 @@ Status KVEngine::SDeleteImpl(Skiplist *skiplist,
     auto hint = hash_table_->GetHint(collection_key);
     std::lock_guard<SpinMutex> lg(*hint.spin);
     Status s =
-        hash_table_->SearchForWrite(hint, collection_key, SortedDataRecord,
-                                    &entry_ptr, &hash_entry, &data_entry);
+        hash_table_->SearchForRead(hint, collection_key, SortedDataRecord,
```

Comment:
SearchForInsert
Search // for read, delete and update

Reply:
rename in other patch